rpcserver.go

```diff
@@ -197,6 +197,10 @@ var (
 			Entity: "onchain",
 			Action: "read",
 		}},
+		"/lnrpc.Lightning/EstimateFee": {{
+			Entity: "onchain",
+			Action: "write",
```

Comment:
This is actually a read operation, no mutation takes place.

Reply:
It is write because it will potentially create a new change address in the btcwallet db. It would require changes to `btcwallet` to avoid this, which is maybe worthwhile?

Reply:
Why does it need to create a new change address? Just to author the txn? If so, I think as a follow up we can add a `dryRun` flag to it. If set, it would do everything _but_ create a new transaction.

Reply:
Created PR to allow this: https://github.com/btcsuite/btcwallet/pull/510

Will update this to use the new API once merged.